services/user.py

```diff
@@ -0,0 +1,43 @@
+from db.models import User
+
+
+def save_user_data(user: User, arguments: dict) -> None:
+    for key, value in arguments.items():
+        if key == "email":
+            user.email = value
+        if key == "first_name":
+            user.first_name = value
+        if key == "last_name":
+            user.last_name = value
+        if key == "username":
+            user.username = value
+        if key == "password":
+            user.set_password(value)
```

Comment:
Can you explain what changed? As for me it will be the same but without loop. With this modification, code is more unreadable
If you want to unify better to do as:
```
for field, value in arguments.items():
    if value is not None:
        if field in ["email", "first_name", "last_name", "username"]:
            setattr(user, field, value)
        elif field == "password":
            user.set_password(value)
```